### PR TITLE
docs: add logos

### PR DESCRIPTION
**Pull request by Doug-AWS**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

Added logos for TypeScript, JavaScript, Java, and .NET to README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ and efficient manner.
 
 The following languages are currently supported:
 
-* Java
-* JavaScript, TypeScript
-* .NET
+* Java ![](docs/src/_images/java.png)
+* JavaScript ![](docs/src/_images/js.png), TypeScript ![](docs/src/_images/ts.png)
+* .NET ![](docs/src/_images/dotnet.png)
```
Comment (eladb) [inline README.md]: Can we find logos with better resolution. This looks pretty crappy

Comment (Doug-AWS) [reply README.md]: It's probably due to scaling as I wanted them all to be no larger than 24 x 24 pixels. That may be too small for the ones with text, such as .NET.

**Comment by eladb** [comment]
@Doug-AWS ping

File: README.md (hunk 1 above)
Comment (Doug-AWS) [reply README.md]: Okay, I bumped them to 40x40 pixels:

image

Comment (eladb) [reply README.md]: Sorry this looks horrible. Maybe we can just put the logos side by side before/after the bulletted list...

**Comment by eladb** [comment]
Also, something is wrong with this PR

**Comment by Doug-AWS** [comment]
I think it's the command I was using to connect my local branch w/origin. I'm closing this and will try a different branch/PR.

**Pull request closed without merging**